src/Illuminate/Broadcasting/BroadcastManager.php

```diff
@@ -21,7 +22,7 @@ class BroadcastManager implements FactoryContract
     /**
      * The application instance.
      *
-     * @var \Illuminate\Contracts\Foundation\Application
+     * @var \Illuminate\Contracts\Container\Container
      */
     protected $app;
```

Comment:
Other than checking for `routesAreCached()`, the only used for `$app` here are related to `Container` contract to resolve `request`, `events`, `queue`, `redis`, `config` and `Psr\Log\LoggerInterface`.